### DISCUSSION TITLE
Unable to use both Copilot and Tabnine simultaneously

### DISCUSSION THREAD

**Discussion by shree675** [community/community#10678] [Copilot Conversations]
I got access to copilot just today, and it is amazing. But I am not able to view the suggestions of both TabNine and Copilot at the same time. Copilot shows its suggestions only if I cancel the TabNine suggestion.

https://user-images.githubusercontent.com/58718144/151503261-dd505abe-ade0-4506-b884-c200aa6dc6b1.mp4

I have also disabled inline suggestions for TabNine. It would be better to view both the suggestions and have more choices.

OS: Linux Mint Cinnamon 20.2
VSCode: 1.63.2

**Comment by katorly** [comment]
I'm using them both, but I didn't encounter this issue. These two just appears at the same time...

**Reply by shree675** [reply]
Hmm, strange. My VSCode is in its latest version. Could it be because of my Linux Mint OS?

**Comment by michaelschufi** [comment]
I have the same problem. Is it a limitation of VS Code? My guess is both extensions are overwriting the suggestions. Especially if inline suggestions are enabled for both. 

Having Copilot's suggestions listed in the "trigger suggestion"-popup, would be great. This way, all the options are presented to the user from both AIs.

**Comment by kouhe3** [comment]
hi,
tabnine [updated](https://www.tabnine.com/blog/announcing-tabnine-next-generation/?utm_source=em&utm_campaign=next-gen-announcement) ,
![image](https://user-images.githubusercontent.com/25522053/173846149-0a2a63fd-87e2-42c5-9330-bd0724e2fbef.png)

if turn on new code-native AI models, I have to use inline completions.

So now Tabnine new models and Copilot are inline completion.

Looks like I'll have to drop either tabnine or github

**Reply by nephi-dev** [reply]
same

**Comment by starball5** [comment]
Related on Stack Overflow: [Can you run Tabnine and Github Copilot together?](https://stackoverflow.com/q/75610875/11107541)
